### PR TITLE
task-54716: changed the the settings title for the new chat push notification

### PR DESCRIPTION
**Pull request by Amine0008**
changed the the settings title for the new chat push notification

### CODE REVIEW

**Review by rdenarie — CHANGES_REQUESTED** [review]
Hello.
I'm not sure the order of words are correct. 
In addition, as this file is already existing in crowdin, your modification will be erased at next syncrhonisation. You should add your modifications directly in crowdin

**Comment by hghorbel** [comment]
> Hello. I'm not sure the order of words are correct. In addition, as this file is already existing in crowdin, your modification will be erased at next syncrhonisation. You should add your modifications directly in crowdin

I'm sorry but i don't understand your remark for the order. The goal is that we add to it as a definition of the action that we can change and update in crowdin if needed to.

**Comment by rdenarie** [comment]
What I see in this pr is a modification of the value of an existing key. 
So, the commit here is not necessary because it will be erased when the sync/merge job will pass.
You only need to make the change in crowdin.

We need a commit in a property file when you add a new key, but as I can see here, you only make modification in exiting key.

**Comment by rdenarie** [comment]
For the order of words :
"Quelqu'un m'envoie ou vers un salle de chat dont je suis membre, un message"
This is not a valid french sentence.

**Comment by rdenarie** [comment]
And this "Someone send me or to a chat room I'm member in a message" is not a valid english sentence

**Comment by hghorbel** [comment]
> And this "Someone send me or to a chat room I'm member in a message" is not a valid english sentence

What do you propose as a pure english sentence ?

**Comment by hghorbel** [comment]
> And this "Someone send me or to a chat room I'm member in a message" is not a valid english sentence

What about: 
EN: I receive a message in a chat room  
FR: Je reçois un message dans une discussion chat 

I think it's much more lighter ?

**Comment by rdenarie** [comment]
It is better yes.
One more time, this change need to occurs in crowdin. No need to make a PR for that

**Comment by rdenarie** [comment]
Hello
As the key are not already in crowdin because not merged in develop, it is ok to change it now in a PR.
Thanks

File: services/src/main/resources/locale/portlet/chat/Resource_fr.properties [hunk 1]
```diff
@@ -261,5 +261,5 @@ Notification.ChatMentionNotificationPlugin.label.footer=Si vous ne souhaitez pas
 UINotification.title.ChatMentionNotificationPlugin=Quelqu'un me mentionne dans le chat
 UINotification.label.ChatMentionNotificationPlugin=Mention dans le chat
 
-UINotification.title.ChatMessageReceivedNotificationPlugin=Quelqu'un m'envoie un message
+UINotification.title.ChatMessageReceivedNotificationPlugin=Je reçois un message dans une discussion chat
 UINotification.label.ChatMessageReceivedNotificationPlugin=Recevoir un message
```
Comment (rdenarie) [inline services/src/main/resources/locale/portlet/chat/Resource_fr.properties]: @hghorbel I find this not homogeneous : 
UINotification.title.ChatMentionNotificationPlugin=Someone mentions me in chat
UINotification.title.ChatMessageReceivedNotificationPlugin=I receive a message in a chat room

I think I prefer the previous formulation completing with "in a chat room": 
UINotification.title.ChatMessageReceivedNotificationPlugin=Someone sends me a message _in a chat room_
What do you think ?

Comment (hghorbel) [reply services/src/main/resources/locale/portlet/chat/Resource_fr.properties]: @rdenarie This is why i suggested the first sentence even if it was complicated to understand and formulate, we do have 2 use cases: 

- I receive a message in one to one discussion (which is a message to me indeed)
- I receive a message in a chat room I'm a member in (the message can be sent to me or to another user in the chat room).

We can just put **Someone sends me a message in a chat room** or just **Someone sends a message in a chat room** (without the word me) to conserve the multiple use cases. 

I prefer the second one without the word me as long as even the one to one discussion is considered as a chat room (Discussion). 
WDYT ?

Comment (rdenarie) [reply services/src/main/resources/locale/portlet/chat/Resource_fr.properties]: I propose : 

- Someone sends me a private message
- Someone sends a message in a chat room

Comment (hghorbel) [reply services/src/main/resources/locale/portlet/chat/Resource_fr.properties]: +1 for Someone sends a message in a chat room and we can change it whenever we want on crowdin.

File: services/src/main/resources/locale/portlet/chat/Resource_fr.properties [hunk 2]
```diff
@@ -261,5 +261,5 @@ Notification.ChatMentionNotificationPlugin.label.footer=Si vous ne souhaitez pas
 UINotification.title.ChatMentionNotificationPlugin=Quelqu'un me mentionne dans le chat
 UINotification.label.ChatMentionNotificationPlugin=Mention dans le chat
 
-UINotification.title.ChatMessageReceivedNotificationPlugin=Quelqu'un m'envoie un message
+UINotification.title.ChatMessageReceivedNotificationPlugin=Je reçois un message dans une discussion chat
```
Comment (rdenarie) [inline services/src/main/resources/locale/portlet/chat/Resource_fr.properties]:
```suggestion
UINotification.title.ChatMessageReceivedNotificationPlugin=Quelqu'un m'envoie un message dans une discussion chat
```

**Review by rdenarie — CHANGES_REQUESTED** [review]

**Comment by rdenarie** [comment]
@hghorbel please check modified file. If ok for you, it is ok for me

**Comment by hghorbel** [comment]
> @hghorbel please check modified file. If ok for you, it is ok for me

Yes it's OK for me

**Review by rdenarie — APPROVED** [review]

**Pull request merged**